src/code/models/node.coffee

```diff
@@ -131,20 +132,23 @@ module.exports = class Node extends GraphPrimitive
   checkIsInIndependentCycle: ->
     visitedNodes = []
     original = this
-    isOwnGrandpa = false
```

Comment:
The main reason I eliminated it was for consistency. I didn't want to add `hasAccumulatorGrandpa` and having two variables that referred to the same concept using different terms seemed potentially confusing. Now that we've eliminated `hasAccumulatorAncestor` (at your suggestion), however, the consistency argument no longer applies so I've restore `isOwnGrandpa`.